protocol/common/feature_flags.go

```diff
@@ -29,4 +29,7 @@ type FeatureFlags struct {
 
 	// EnableCelerBridge indicates whether we should enable the Celer bridge in the Router
 	EnableCelerBridge bool
+
+	// EnableMercuryoProvider indicates whether we should enable the Mercuryo provider in the Wallet
+	EnableMercuryoProvider bool
```

Comment:
Why do we need both feature flag and config field? 🤔

Reply:
I'm not familiar with the structure, i just did the same thing we had for EnableCelerBridge. should i remove config field?

Reply:
Looks to me that we want this to be disabled forever on build-time, so yes. But I'm also not familiar with this code, let's ping @dlipicar, hopefully he can shed some light 🙂

Reply:
Hey @igor-sirotin @dlipicar if there is no issue with current changes, i'm going to merge it this evening, I already got approval from QA.

Reply:
for this particular PR, it doesn't look like storing the feature flag brings any benefit since we're not using it anyhow after login and it's not something we'll toggle mid-session, so we can just use the config field to initialize the list of providers. We can add the feature flag in the future if we integrate some sort of location detection or user acknowledgement that they're not from the UK.